scripts.js

```diff
@@ -4,19 +4,12 @@ function add_doggo() {
         .then(show_image);
 }
 
+function show_image({message}) {
+    const current_image = document.getElementById("static-doggo");
+    current_image.src = message
+// function show_image(dogObj) {
```

Comment:
You left the old code in here commented out.

Reply:
Yeah, I did that on purpose. I didn't want to get rid of it, in case I look back on this code 3 months down the line and forget what what I did lol. I'm sure it won't be a problem then, but just to be sure, I'll leave it there.

Reply:
Ah.  That's a good time to use a comment to explain what you're doing, and be descriptive in the commit message to help explain it.  `git blame` lets you look back at a line of code and see when it was changed, so it's easy to jump back and understand why a line is how it is.

Reply:
fixed.

Reply:
I meant you can delete the commented out code, and easily find it again using git history, and git blame.  No need to leave it and comment about git blame.